package.json

```diff
@@ -53,6 +53,8 @@
     "export": "node ./scripts website export",
     "build-tests": "rollup -c ./rollup.test.config.js",
     "test": "node ./scripts test",
+    "test:unit-browser": "npm run test -- -s unit -p 8080 -l -c ",
+    "test:visual-browser": "npm run test -- -s visual -p 8080 -l -c ",
```

Comment:
`-c` flag is already taken

Reply:
make it `--coverage`

Reply:
this is the -c flag to append chrome/firefox is not the coverage flag

Reply:
so `-c chrome,firefox`